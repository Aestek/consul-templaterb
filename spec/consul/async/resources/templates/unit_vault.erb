<% ['users','groups'].each do |entity_type|
%><%= entity_type.capitalize %>:
<%
    secrets("auth/ldap/#{entity_type}/").each do |entity|
%> * <%=entity%>: <%= secret("auth/ldap/#{entity_type}/#{entity}")['data'].to_json %>
<% end %>
<% end %>
