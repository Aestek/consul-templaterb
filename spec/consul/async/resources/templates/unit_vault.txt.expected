Users:
 * d.vador: {"groups":"","policies":["empire_endor_admin"]}

Groups:


